cpp/ps0000/ps0000soln.cpp

```diff
@@ -1,8 +1,9 @@
-#include <iostream>
+#include <iostream> // 2/2
 using namespace std;
 
-int main(void)
+int main(void) // 1/1
 {
-  cout << "Hello, World!" << "\n";
-  return 0;
-}
\ No newline at end of file
+  cout << "Hello, World!" << "\n"; // 4/4
+  return 0; // 1/1
+}
+// great work 😎 10/10 ⭐

```

Comment:
- [x] most of code errors and bugs are fixed
- [x] +1 mark for your first commit